pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -117,11 +116,6 @@ func checkEvents(t *testing.T, fr *record.FakeRecorder, testName string, wantEve
 	return eventFromChannel(fr.Events, testName, wantEvents)
 }
 
-func checkCloudEvents(t *testing.T, fce *cloudevent.FakeClient, testName string, wantEvents []string) error {
```

Comment:
The reason for this is that when cloud events for taskruns were introduced in v0.14, they were meant to add cloudevents for pipelinerun too. This code is actually used https://github.com/tektoncd/pipeline/pull/2938/files... so I'll rebase that PR now to include it.